project/AkkaDisciplinePlugin.scala

```diff
@@ -21,7 +21,7 @@ object AkkaDisciplinePlugin extends AutoPlugin with ScalafixSupport {
   override def requires: Plugins = JvmPlugin && ScalafixPlugin
   override lazy val projectSettings = disciplineSettings
   
-  val strictProjects = Set("akka-discovery")
+  val strictProjects = Set("akka-discovery", "akka-stream-typed", "akka-cluster-typed")
```

Comment:
The idea originally was everything was strict and this qualification would not be needed, so eventually, that's the goal.

Reply:
But that work is a module by module process :)